spec/rspec/support/object_inspector_spec.rb

```diff
@@ -92,6 +92,17 @@ def with_date_loaded
             matcher_without_a_description.inspect)
         end
       end
+
+      context "with double objects" do
+        it "displays double name" do
+          double = double('Foo')
+          allow(double).to receive(:name).and_return("Foo")
+          ObjectInspector.register double.class do |double|
```

Comment:
FYI your specs are failing on Travis because the `double` here shadows / overwrites the existing local variable in this scope which is considered "less than ideal" practise and we have rubocop checking for it :)